department-of-veterans-affairs/va.gov-cms
# VACMS-000: Remove ie unfriendly presence check and replace with standard old school check

## Description
Removes `?.` operator from `docroot/modules/custom/va_gov_backend/js/script.js` and replaces with old school check to keep ie happy.

## Testing done
Drupal cache & page cache clear.

## Screenshots
N/A

## QA steps
As an administrator, go to `node/add/campaign_landing_page`, open the spotlight tab, and click the boolean toggle.
Visually verify that `LINKS WITH SUMMARIES` has the red asterisk.

### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
